From 2e732edae699b87eb067a1cda5f20b0e0f54e488 Mon Sep 17 00:00:00 2001
From: "Badrappan, Jeevaka" <jeevaka.badrappan@intel.com>
Date: Tue, 19 Sep 2023 11:21:47 +0530
Subject: [PATCH] [REVERT ME] kernel: Use firmware version available in
 upstream firmware

As the latest firmwares are not available in upstream firmware git,
updated the firmware versions to the ones available.

Tracked-On: OAM-112224
Signed-off-by: Badrappan, Jeevaka <jeevaka.badrappan@intel.com>
---
 drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c b/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
index 577f4170bf71..4abee26cb0f0 100644
--- a/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
+++ b/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
@@ -87,13 +87,13 @@ void intel_uc_fw_change_status(struct intel_uc_fw *uc_fw,
 #define INTEL_GUC_FIRMWARE_DEFS(fw_def, guc_maj, guc_mmp) \
 	fw_def(METEORLAKE,   0, guc_mmp(mtl,  70, 6, 8)) \
 	fw_def(PONTEVECCHIO, 0, guc_mmp(pvc,  70, 7, 0)) \
-	fw_def(DG2,          0, guc_mmp(dg2,  70, 7, 0)) \
+	fw_def(DG2,          0, guc_maj(dg2,  70, 5, 1)) \
 	fw_def(XEHPSDV,  0, guc_mmp(xehpsdv,  70, 7, 0)) \
-	fw_def(ALDERLAKE_P,  0, guc_mmp(adlp, 70, 7, 0)) \
+	fw_def(ALDERLAKE_P,  0, guc_maj(adlp, 70, 5, 1)) \
 	fw_def(ALDERLAKE_S,  0, guc_mmp(tgl,  70, 7, 0)) \
 	fw_def(DG1,          0, guc_mmp(dg1,  70, 7, 0)) \
 	fw_def(ROCKETLAKE,   0, guc_mmp(tgl,  70, 7, 0)) \
-	fw_def(TIGERLAKE,    0, guc_mmp(tgl,  70, 7, 0)) \
+	fw_def(TIGERLAKE,    0, guc_maj(tgl,  70, 5, 1)) \
 	fw_def(JASPERLAKE,   0, guc_mmp(ehl,  70, 7, 0)) \
 	fw_def(ELKHARTLAKE,  0, guc_mmp(ehl,  70, 7, 0)) \
 	fw_def(ICELAKE,      0, guc_mmp(icl,  70, 7, 0)) \
-- 
2.40.0

